build.sbt

```diff
@@ -87,15 +87,15 @@ lazy val parsing = project("akka-parsing")
 
 lazy val httpCore = project("akka-http-core")
   .dependsOn(parsing)
-  .addAkkaModuleDependency("akka-stream", "provided")
```

Comment:
this is what breaks the 10.0 dependency semantics -- we moved to "explicitly depend on akka" in 10.1, however this should not be backported to 10.0